app/src/main/java/org/mozilla/fenix/compose/ListItemTabLarge.kt

```diff
@@ -89,18 +97,24 @@ fun ListItemTabLarge(
  * ```
  *
  * @param imageUrl URL from where the to download a header image of the tab this composable renders.
+ * @param onClick Optional callback to be invoked when this composable is clicked.
+ * @param backgroundColor Background [Color] of the item.
  * @param title Composable rendering the title of the tab this composable represents.
  * @param subtitle Optional tab caption composable.
- * @param onClick Optional callback to be invoked when this composable is clicked.
  */
 @Composable
 fun ListItemTabLarge(
     imageUrl: String,
     onClick: () -> Unit,
+    backgroundColor: Color = FirefoxTheme.colors.layer2,
```

Comment:
Let's move this before `onClick`. In terms of how I envision we order composable parameters, it would be stateful parameters (String, Color, etc.), event handlers and lastly composables.